provision-osx/tasks/configure_base_system.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+-
+  stat:
+    path: ~/.bash_profile
+  register: bash_file
+
+-
+  name: Create user bash file
+  file:
+    path: ~/.bash_profile
+    state: touch
+  when: bash_file.stat.exists == False
+
+-
+  name: Configure bash profile language encoding
+  lineinfile:
+    dest: ~/.bash_profile
+    line: "export LANG={{ buildfarm_lang_env_var }}"
+
+-
+  name: Configure profile file
+  lineinfile:
+    dest: ~/.profile
+    line: "source ~/.bash_profile"
```

Comment:
@odra ok, can you create a PR to remove it? If we are going to include this, we need to make sure it won't cause circular dependencies between those files.